### PR TITLE
Use nodes as services instead of controllers

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

**Comment by julienw** [comment]
There are 2 commits: 
- first one is simply adding a `_` in front of some variable names in the matching code to shush some warnings. We still have warnings about unused Result but this would be for another PR;
- second one is the actual change. Too bad we can't change an existing service yet, we would need to delete it and add it again with all its getters/setters. I don't want to do this work now and I filed https://github.com/fxbox/taxonomy/issues/97 instead.

File: src/lib.rs [hunk 1]
```diff
@@ -348,24 +348,35 @@ impl OpenzwaveAdapter {
...
+                    }
+                    ZWaveNotification::NodeNaming(_node)             => {
+                        // unfortunately we can't change a service' properties :(
+                        // https://github.com/fxbox/taxonomy/issues/97
+                        // When it's done we can move the properties change from above to here.
                     }
-                    ZWaveNotification::NodeNew(_node)               => {}
-                    ZWaveNotification::NodeAdded(_node)             => {}
                     ZWaveNotification::NodeRemoved(_node)           => {}
                     ZWaveNotification::ValueAdded(vid)              => {
```
Comment (dhylands) [inline src/lib.rs:372]: I think that we need to wait longer than this. There are still a bunch of node queries which haven't been done yet.

Certainly, by the time AllAwakeNodesQueried(?sp) then we could advertise. Otherwise commands sent too early will fail.

I'd probably create an issue and leave that for a different PR.

**Comment by dhylands** [comment]
r+ lgtm

File: src/lib.rs (hunk 1 above)
Comment (julienw) [inline src/lib.rs:372]: I agree we might need some form of bufferisation; but first I'd like to understand which problems we have now :)
